example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -0,0 +1,273 @@
+    const collectionJson = JSON.parse(fs.readFileSync(`${collectionsDir}/s3_MOD09GQ_006.json`, 'utf8'));
+    collectionJson.duplicateHandling = 'error';
+    const collectionData = Object.assign({}, collectionJson, {
+      name: collection.name,
+      dataType: collectionJson.dataType + testSuffix
+    });
+
+    const providerJson = JSON.parse(fs.readFileSync(`${providersDir}/s3_provider.json`, 'utf8'));
+    const providerData = Object.assign({}, providerJson, {
+      id: provider.id,
+      host: config.bucket
+    });
+
+    // populate collections, providers and test data
+    await Promise.all([
+      uploadTestDataToBucket(config.bucket, s3data, testDataFolder),
+      apiTestUtils.addCollectionApi({ prefix: config.stackName, collection: collectionData }),
+      apiTestUtils.addProviderApi({ prefix: config.stackName, provider: providerData })
+    ]);
+
+    const inputPayloadJson = fs.readFileSync(inputPayloadFilename, 'utf8');
+    // update test data filepaths
+    inputPayload = await setupTestGranuleForIngest(config.bucket, inputPayloadJson, granuleRegex, testSuffix, testDataFolder);
+    const granuleId = inputPayload.granules[0].granuleId;
+
+    expectedPayload = loadFileWithUpdatedGranuleIdPathAndCollection(templatedOutputPayloadFilename, granuleId, testDataFolder, newCollectionId);
+    expectedPayload.granules[0].dataType += testSuffix;
+
+    workflowExecution = await buildAndExecuteWorkflow(
+      config.stackName,
+      config.bucket,
+      workflowName,
+      collection,
+      provider,
+      inputPayload,
+      {
+        cmrFileType: 'ummg1.4',
+        additionalUrls: [cumulusDocUrl]
+      }
+    );
+  });
+
+  afterAll(async () => {
+    // clean up stack state added by test
+    await Promise.all([
+      deleteFolder(config.bucket, testDataFolder),
+      collectionModel.delete(collection),
+      providerModel.delete(provider),
+      executionModel.delete({ arn: workflowExecution.executionArn }),
+      granulesApiTestUtils.deleteGranule({
+        prefix: config.stackName,
+        granuleId: inputPayload.granules[0].granuleId
+      })
+    ]);
+  });
+
+  xit('completes execution with success status', () => {
+    expect(workflowExecution.status).toEqual('SUCCEEDED');
+  });
+
+  // This is a sanity check to make sure we actually generated UMM and also
+  // grab the location of the UMM file to use when testing move
+  describe('the processing task creates a UMM file', () => {
+    let processingTaskOutput;
+    let ummFiles;
+
+    beforeAll(async () => {
+      processingTaskOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'FakeProcessing');
+
+      ummFiles = processingTaskOutput.payload.filter((file) => file.includes('.cmr.json'));
+      s3ummJsonFileLocation = ummFiles[0];
+    });
+
+    it('creates a UMM JSON file', () => {
+      expect(ummFiles.length).toEqual(1);
+
+      const xmlFiles = processingTaskOutput.payload.filter((file) => file.includes('.cmr.xml'));
```

Comment:
I'd put this in a separate it
`it('does not create any cmr.xml files')`

Reply:
That's fine, but this all more of a sanity check to make sure we set up the test correctly.